config/settings_prod.py

```diff
@@ -154,27 +154,21 @@
 # this guard ensures that only OSIDB running in MPP logs to filesystem
 # TODO: Remove after OSIDB is fully migrated to MPP
 if get_env("MPP", is_bool=True, default="False"):
-    LOGGING.update(
-        {
-            "handlers": {
-                "celery": {
-                    "class": "logging.handlers.RotatingFileHandler",
-                    "formatter": "verbose_celery",
-                    "filename": "/var/log/prod-celery.log",
-                    "maxBytes": LOG_FILE_SIZE,
-                    "backupCount": LOG_FILE_COUNT,
-                },
-                "console": {
-                    "level": "INFO",
-                    "class": "logging.handlers.RotatingFileHandler",
-                    "formatter": "verbose",
-                    "filename": "/var/log/prod-django.log",
-                    "maxBytes": LOG_FILE_SIZE,
-                    "backupCount": LOG_FILE_COUNT,
-                },
-            }
-        }
-    )
+    LOGGING["handlers"]["celery"] = {
```

Comment:
Sorry for the dump question, but what was the problem in the original code?

Reply:
Basically the previous code turned
```python
{
 "loggers":
    {
      "django": ...,
      "bzimport" ...,
      ...
    }
}
```

into

```python
{
 "loggers":
    {
      "bugzilla": ...,
    }
}
```

All other loggers were overwritten with bugzilla logger.